acceptance/test/smart-proxy/features/list.feature

```diff
@@ -1,5 +1,9 @@
 Feature: List Namespaces
 
-  Scenario: user list namespace
+  Scenario: user list namespaces
     Given User has access to "10" namespaces
     Then the User can retrieve only the namespaces they have access to
+
```

Comment:
I understand you had some issues with duplicate namespaces, that would be a good scenario to add

Reply:
that's true, I'll work at that in another PR